### ISSUE TITLE
[Bug]: Partner cannot change effect their neighbourhood

### ISSUE THREAD

**Issue by aaaaargZombies** [geeksforsocialchange/PlaceCal#2046]
When you add a partners address they are automatically assigned a neighbourhood

This is a problem now we have two neighborhood datasets especially

If they do not feel like they belong to that neighbourhood , or if there no placecal site for them but one near by they have no way to change it. 

If we let them control their own neighbourhood they could appear on sites that a neighbourhood admin does not want them to.

**Comment by kimadactyl** [comment]
> If they do not feel like they belong to that neighbourhood

Imo this is tough - the neighbourhoods are set by HM government, not us, it's kinda akin to letting people choose their postcode if they don't like the one they have to me. I would call this way outside the scope of what PlaceCal is about.

**Comment by katjam** [comment]
I disagree with closing this. There was a case that @honor-gfsc brings up where a Partner's address is just outside an existing PlaceCal instance and we think they whould be able to be manually included (maybe by chosing a different neighbourhood). Though this could be a neighbourhood admin making this change rather than letting partners do it themselves.

Part of the issue here is that it's articulated more as a solution "Partner should be able to choose their neighbourhood" rather than the actual problem which is that sometimes partners who should be part of a place based PlaceCal are not and we ought to provide a way of addressing this.

@aaaaargZombies was there a particular case you came across regarding the double neighbourhood dataset?

**Comment by aaaaargZombies** [comment]
So this isn't as bad as I first thought because I believe a Partner can opt in via "service area". This is a bit messy for people though, it's opaque for partners which data set a neighborhood based site is drawing from, they also need to understand about wards and that we have two definitions of wards.

But the case still stands that as boundaries change people who identify with one region may find themselves disconnected. I think this is a loose end resulting from the addition of the new dataset. There's probably some work to figure out how to smooth this transition still? There's a couple things that need to line up to make sure people don't fall through the cracks.

The title of the bug may not be very good / accurate.

**Comment by kimadactyl** [comment]
All the issues you've stated are real and I think covered in other places or will come up again.

But I just don't think the solution to it is to basically let people go "I don't identify with living in the M15 postcode, I feel like I live in M14". It's kinda nonsensical to me. We need to deal with this in other ways.
